PalasoUIWindowsForms/Reporting/MemoryManagement.cs

```diff
@@ -37,7 +37,9 @@ public static class MemoryManagement
 		public static bool CheckMemory(bool minor, string eventDescription, bool okToBotherUser)
 		{
 			var heapMem = GC.GetTotalMemory(true); // first, as it may reduce other numbers
+			var is64BitProcess = IntPtr.Size == 8; // according to MSDN
 			long memorySize64;
```

Comment:
These variable names seem to indicate that they are used only when working with a 64bit process but that doesn't seem to be the case. Is the 64 after memorySize and workingSet correct or helpful?